docs/api.md

```diff
@@ -99,3 +99,38 @@ You can find sample requests/responses [here](./api-samples/v1/import.md)
 | ----- | -------- |
 | 200   | |
 | 500 | something went wrong (see logs of the simulator server) |
+
+## Watch the simulator's resources
+
+Monitor simulator resources and get updates automatically.
```

Comment:
Need to describe about server-side events so that client can know how to get update.